### PR TITLE
Allow disabling terrascan

### PR DESCRIPTION
**Pull request by JonZeolla**
# Contributor Comments
Allow disabling `terrascan`

## Pull Request Checklist
Thank you for submitting a contribution to `goat`!

In order to streamline the review of your contribution we ask that you review and comply with the below requirements:
- [X] Rebase your branch against the latest commit of the target branch, which likely should be `main`
- [X] If there is an issue associated with your Pull Request, align your PR title with [this documentation](https://help.github.com/en/articles/closing-issues-using-keywords)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,14 +2,18 @@
 The Grand Opinionated AutoTester (GOAT) automatically applies Seiso's standard testing.
 
 ## Getting Started
-1. Create a dictionary text file in `.github/etc/dictionary.txt` (relative to the root of your git repo).
+1. Create a per-repository dictionary (relative to the root of your git repo).
+```bash
+$ mkdir -p .github/etc/
```
Comment (tpears) [inline README.md]: 7 & 8 break the markdown linter: `Dollar signs used before commands without showing output`

File: README.md [hunk 2]
```diff
@@ -28,16 +32,32 @@ jobs:
...
 ```
+1. Configure the goat to skip terrascan scanning.
+```bash
+uses: seisollc/goat@v0.4.0
+with:
+  disable_terrascan: true
+```
+1. Provide a linting configuration for any of the linters `super-linter` uses.
+```bash
+$ mkdir -p .github/linters/
```
Comment (tpears) [inline README.md:54]: Would expect 54 & 55 to throw the linter error as in 7 & 8 but they were not in the log.

Comment (tpears) [reply README.md:54]: It only throws the error when all lines in a block begin with the `$`. Markdown style guidelines advise against using `$` unless you're using it to distinguish between entered commands and their output.

https://github.com/markdownlint/markdownlint/commit/0ec5e846be70641641aaf670c48dfe7972106e96
\<removed dead link\>

**Review by tpears — CHANGES_REQUESTED** [review]
Remove `$` from bash code blocks that don't need to differentiate between commands and their output.

**Comment by JonZeolla** [comment]
Sorry for that @tpears I got green on d9bfc42 and didn't expect my follow-on commits to cause an issue.  Should be good to go now

**Review by tpears — APPROVED** [review]

**Pull request merged**